internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,465 @@
+package asyncapi
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/antihax/optional"
+	"github.com/go-yaml/yaml"
+	"github.com/iancoleman/strcase"
+	"github.com/spf13/cobra"
+	spec2 "github.com/swaggest/go-asyncapi/spec-2.4.0"
+	yaml3 "k8s.io/apimachinery/pkg/util/yaml"
```

Comment:
Why are we importing two separate yaml packages? Isn't github.com/go-yaml/yaml enough?

Reply:
Importing "k8s.io/apimachinery/pkg/util/yaml" for ToJSON() function to convert schema payload to JSON.

Reply:
Can we convert YAML --> `map[string]any` --> JSON, without needing to rely on this library?